streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/resource/ServiceBundleResource.java

```diff
@@ -126,6 +127,24 @@ public Response addServiceBundle(ServiceBundle serviceBundle) throws IOException
         }
     }
 
+    /**
+     * Add a new service bundle.
+     */
+    @PUT
```

Comment:
1. Let's be consistent:

PUT - create
POST - update

We may need to change bootstrap script as well for that. 

2. javadoc description looks like wrong since this is for update, not for create.
3. OK instead of CREATED

Reply:
I have updated the javadoc and the response message for the API. From the discussion I had with arun and satish I think we use PUT for update and POST for create.

Reply:
Oh OK. I was confused and you're right on HTTP method.